integration/docker/search_test.go

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2017 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package docker
+
+import (
+	. "github.com/clearcontainers/tests"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("search", func() {
+	var (
+		args []string
+	)
+
+	Context("search an image", func() {
+		It("should filter the requests", func() {
+			args = []string{"--filter", "is-official=true", "--filter=stars=3", Image}
```

Comment:
I guess currently stars is 3 but what will happen if tomorrow is 4 o 2? does this test fail if the stars of the image change?